Source/VMware.vSphereDSC/DSCResources/VMHostNetwork/VMHostVss/VMHostVss.ps1

```diff
@@ -107,16 +85,17 @@ class VMHostVss : VMHostBaseDSC {
     [VMHostVss] Get() {
         Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
 
+        $result = [VMHostVss]::new()
+        $result.Server = $this.Server
+
         $this.ConnectVIServer()
         $vmHost = $this.GetVMHost()
+        $this.GetNetworkSystem($vmHost)
 
-        $result = [VMHostVss]::new()
-        $result.Server = $this.Server
         $result.Name = $vmHost.Name
-
         $this.PopulateResult($vmHost, $result)
 
-        if ($null -ne $result.Key) {
+        if ('' -ne $result.Key) {
```

Comment:
Change to [string]::Empty and modify the if/else clause:

```
 $result.Ensure = if ($result.Key -ne [string]::Empty) { 'Present' } else { 'Absent' }
```

Reply:
Changed